05-extra.md

```diff
@@ -0,0 +1,225 @@
+---
+layout: page
+title: Version Control with Git
+subtitle: Additional Topics
```

Comment:
Ideally, they should: running the validator (`python tools/check.py`) kicks back an error about the missing header field. See #26, #27, #28, #29 .